src/stream/src/executor/backfill/no_shuffle_backfill.rs

```diff
@@ -403,22 +405,6 @@ where
                     upstream_chunk_buffer.clear()
                 }
 
-                self.metrics
-                    .backfill_snapshot_read_row_count
-                    .with_label_values(&[
-                        upstream_table_id.to_string().as_str(),
-                        self.actor_id.to_string().as_str(),
-                    ])
-                    .inc_by(cur_barrier_snapshot_processed_rows);
-
-                self.metrics
-                    .backfill_upstream_output_row_count
-                    .with_label_values(&[
-                        upstream_table_id.to_string().as_str(),
-                        self.actor_id.to_string().as_str(),
-                    ])
-                    .inc_by(cur_barrier_upstream_processed_rows);
-
```

Comment:
If we remove this, we won't report the metrics until backfill is complete.

We can instead report the metrics change at the part where we do `break 'backfill_loop` in `snapshot_read`, since that's the only place we miss the metrics reporting.

https://github.com/risingwavelabs/risingwave/blob/5163877eae7a65eb13e377316acd55a596ed9f8a/src/stream/src/executor/backfill/arrangement_backfill.rs#L304

https://github.com/risingwavelabs/risingwave/blob/5163877eae7a65eb13e377316acd55a596ed9f8a/src/stream/src/executor/backfill/no_shuffle_backfill.rs#L305